cds/modules/deposit/static/js/cds_deposit/avc/components/cdsForm.js

```diff
@@ -183,11 +183,14 @@ function cdsFormCtrl($scope, $http, $q, schemaFormDecorators) {
     function(data, query) {
       var userInput = authorFromUser(query);
       var suggestions = data.data.map(function (author) {
-        var fullName = (author.lastname || '') + ', ' +
-                       (author.firstname || '');
-        var valueObj = {
-          name: fullName
-        };
+        var valueObj = {};
+
+        if (author.firstname) {
+          valueObj.fullName = (author.lastname || '') + ', ' +
+            (author.firstname || '');
+        } else {
+          valueObj.fullName = (author.name) || '';
```

Comment:
hmm shouldn't be ``valueObj.name`` instead of ``valueObj.fullName`` ? https://github.com/CERNDocumentServer/cds/blob/cdslabs_qa/cds/modules/deposit/static/templates/cds_deposit/angular-schema-form/uiselect.html#L30